Code/Tools/TestImpactFramework/Runtime/Native/Code/Source/TestEngine/Native/TestImpactNativeTestEngine.cpp

```diff
@@ -65,7 +65,7 @@ namespace TestImpact
 
         if (jobInfo.GetLaunchMethod() == LaunchMethod::StandAlone)
         {
-        
+
```

Comment:
nit: Could we remove this new line break?

Reply:
Yeah I'll remove this.